packages/forms-web-app/src/config.js

```diff
@@ -66,8 +66,10 @@ module.exports = {
 				description: 'description',
 				email: 'email',
 				hasInterestedPartyNo: 'hasInterestedPartyNo',
+				comment: 'comment',
 				id: 'id',
```

Comment:
Are these values being used. Are we not moving away from this config?

Reply:
addressed in next PR for removal